src/crewai/agent.py

```diff
@@ -608,6 +618,26 @@ def _render_text_description(self, tools: List[Any]) -> str:
 
         return description
 
+    def _inject_date_to_task(self, task):
+        """Inject the current date into the task description if inject_date is enabled."""
+        if self.inject_date:
+            from datetime import datetime
+            try:
+                valid_format_codes = ['%Y', '%m', '%d', '%H', '%M', '%S', '%B', '%b', '%A', '%a']
```

Comment:
Do we also want to include `%Z` for timezone?

Reply:
foollow up